faq/dedibox-in-scaleway-console.mdx

```diff
@@ -24,99 +25,102 @@ Once the process is complete, you will be able to manage your Dedibox Servers an
   * outsourced services or if you are outsourcing your services
 </Message>
 
-## What happens during the account linking process?
+### What happens during the account linking process?
 
-During the process, your Dedibox account is linked to your Scaleway account so that you can find all your Dedibox servers in the [new console](https://console.scaleway.com).<br />
-When you do not have a Scaleway account, the process allows you to create a Scaleway account from your Dedibox information. This allows you to find all of your Dedibox servers in the new console and enjoy a new experience.
+During the process, your Dedibox account is linked to your Scaleway account so that you can find all your Dedibox servers in the [new console](https://console.scaleway.com). When you do not have a Scaleway account, the process allows you to create a Scaleway account from your Dedibox information. This allows you to find all of your Dedibox servers in the new console and enjoy a new experience.
```

Comment:
```suggestion
During the process, your Dedibox account is linked to your Scaleway account so that you can find all your Dedibox servers in the [new console](https://console.scaleway.com).
If are new to Scaleway, the process allows you to create a Scaleway account from your Dedibox information. You can then find all your Dedibox servers in the Scaleway console and enjoy a new experience.
```